lib/services/northBound/deviceGroupAdministrationServer.js

```diff
@@ -268,20 +268,40 @@ function loadContextRoutes(router) {
         restUtils.checkRequestAttributes('headers', mandatoryHeaders),
         restUtils.checkBody(templateGroup),
         handleCreateDeviceGroup);
+    
+    router.post('/iot/cgroups',
```

Comment:
Originally I saw this implemented in a more compact way, i.e. using

```` 
router.post('/iot/:services?/:cgroups?'
```

Why this was changed? It's ok also in the current way, but I'm curious about it...

Reply:
(Btw, after some internal discussion, we probably would suggest a different literal for the URL? cgroups -> configGroups. However, please wait for confirmation before start doing the changes in the PR).

Reply:
This was changed to increase the test coverage which was decreased previously. OK will wait for your confirmation whether to keep cgroups or configGroups.

Reply:
I do confirm the chosen token is `configGroups`. Thanks!

Reply:
Fixed at commit e6cee6a